terraform/deployments/cluster-services/.tflint.hcl

```diff
@@ -0,0 +1,13 @@
+config {
+  varfile = [
+    "../variables/common.tfvars",
+    "../variables/test/common.tfvars",
+  ]
+}
+
+plugin "aws" {
+  enabled = true
+  version = "0.25.0"
```

Comment:
optional: I don't suppose there's a way for us to avoid pinning this version number in every module's linter config?

Reply:
fr don't feel you need to address this in this PR btw — I appreciate we're already at least 2 yaks deep here 😅

Reply:
Ahah yeah - I might come back to this. There wasn't an obvious way in the docs.

Reply:
no probs at all — thanks for looking!